Zilon.Core/Zilon.Core/StaticObjectModules/LifetimeModule.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+using Zilon.Core.Tactics;
+
+namespace Zilon.Core.StaticObjectModules
+{
+    public class LifetimeModule : ILifetimeModule
+    {
+        private readonly IStaticObjectManager _staticObjectManager;
+        private readonly IStaticObject _parentStaticObject;
+        private readonly IPropDepositModule _propDepositModule;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_propDepositModule'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5383584)